tests/README.md

```diff
@@ -0,0 +1,94 @@
+# Unit and regression tests for Snakemake
```

Comment:
Sorry, one final thing besides my comment above: can you merge the content of this readme into https://snakemake.readthedocs.io/en/stable/project_info/contributing.html#testing-guidelines, and just put a link to that doc section into this readme? This way, tests are documented in just one place but the readme still helps people who haven't found the section in the online documentation.

Reply:
Yes, will do

Reply:
Content was merged, so we no longer have redundant docs.

One question - the docs mention wercker.com. Is this still a thing? I thought we were all running of GitHub actions now?